src/api/nodeRpcCall.js

```diff
@@ -428,7 +428,7 @@ const stakeToPolkaBondAndExtra = async (payload, callback) => {
   const api = await getApi();
   const transferExtrinsic = isUserHavePolkaStake
     ? await api.tx.staking.bondExtra(dollarsToGrains(amount))
-    : await api.tx.staking.bond(walletAddress, dollarsToGrains(amount), 'Staked');
```

Comment:
API changed in Runtime [v11.0.3](https://github.com/liberland/liberland_substrate/releases/tag/v11.0.3).